x-pack/plugins/apm/public/components/alerting/chart_preview/index.tsx

```diff
@@ -102,18 +110,20 @@ export function ChartPreview({
           position={Position.Left}
           tickFormat={yTickFormat}
           ticks={5}
-          domain={{ max: yMax, min: NaN }}
```

Comment:
`min: NaN` looks super odd. Glad to see that gone. 
I recall that we needed to expand the range, in case the `threshold` was significantly above the default domain. Is that no longer the case?

Reply:
Created issue #143659 to address this.